core/src/main/java/hudson/Functions.java

```diff
@@ -851,6 +851,10 @@ public static String getFooterURL() {
         return JobPropertyDescriptor.getPropertyDescriptors(clazz);
     }
 
+    public static List<JobPropertyDescriptor> getJobPropertyDescriptors(Job job) {
+        return DescriptorVisibilityFilter.apply(job, JobPropertyDescriptor.getPropertyDescriptors(job.getClass()));
+    }
```

Comment:
Because `Functions` is found to `f` in Jelly, which is how this is being used.

Really it would make sense to kill the whole `Functions` class, but that presumes a more expressive way of writing logic in views to begin with. (Invoking static methods in core is painful but possible from JEXL; in plugins, all but impossible.) Cf. JENKINS-31203.